.github/workflows/ubuntu-ci.yml

```
@@ -0,0 +1,71 @@
+
+name: Fast DDS Discovery Server Ubuntu CI
+
+on:
+  workflow_dispatch:
+    inputs:
+      os-version:
+        description: 'OS version to run the workflow'
+        required: false
+        default: 'ubuntu-22.04'
+        type: string
+      colcon-args:
+        description: 'Extra arguments for colcon cli'
+        required: false
+        type: string
+      cmake-args:
+        description: 'Extra arguments for cmake cli'
+        required: false
+        type: string
+      ctest-args:
+        description: 'Extra arguments for ctest cli'
+        required: false
+        type: string
+      discovery-server-branch:
+        description: 'Branch or tag of Discovery Server repository'
+        type: string
+        required: true
+      fastdds-branch:
+        description: 'Branch or tag of Fast DDS repository'
+        type: string
+        required: true
+      run-tests:
+        description: 'Run test suite of Fast DDS Discovery Server'
+        required: false
+        type: boolean
+        default: true
+      use-ccache:
+        description: 'Use CCache to speed up the build'
+        required: false
+        type: boolean
+        default: false
+
+  pull_request:
+    types:
+      - review_requested
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - '!**/CMakeLists.txt'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  ubuntu-ci:
+    if: ${{ !(github.event_name == 'pull_request') || !contains(github.event.pull_request.labels.*.name, 'conflicts') || false }}
```

I don't think this is necessary
```suggestion
    if: ${{ !(github.event_name == 'pull_request') || !contains(github.event.pull_request.labels.*.name, 'conflicts') }}
```